app/src/main/java/simform/gitexcercise/android/validation/Validator.kt

```diff
@@ -0,0 +1,31 @@
+package simform.gitexcercise.android.validation
+
+object Validator {
+
+    object Password {
+        const val MIN_LENGTH = 8
+        const val MAX_LENGTH = 16
+    }
+
+    enum class ValidCredentials {
+        VALID, INVALID_EMAIL, INVALID_PASSWORD
+    }
+
+    fun isValidCredentials(email: String, password: String): ValidCredentials {
+        return if (isValidEmail(email)) {
```

Comment:
Simplify the return using guard statements

fun isValidCredentials(email: String, password: String): String {
        if (!isValidEmail(email)) return ValidCredentials.INVALID_EMAIL
        if (!isValidPassword(email)) return ValidCredentials.INVALID_PASSWORD
            ...
    }

Reply:
okay will follow it